datafusion/expr/src/logical_plan/plan.rs

```diff
@@ -1919,14 +1947,11 @@ impl Join {
 pub struct Subquery {
     /// The subquery
     pub subquery: Arc<LogicalPlan>,
+    /// The outer references used in the subquery
+    pub outer_ref_columns: Vec<Expr>,
```

Comment:
👍 This field can help to check if it is a correlated subquery.

Reply:
Yes.